lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,17 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( 'edit' === $context ) {
```

Comment:
Okay, that's how I originally had it. I thought that since it only ever shows up in edit context, it should be in that conditional.  Will revert it.